### ISSUE TITLE
Ah yea you are right, will be fixed in the next release

### ISSUE THREAD

**Issue by madilynh343** [RedShyGuy/Vapecord-ACNL-Plugin#103]
Ah yea you are right, will be fixed in the next release

_Originally posted by @RedShyGuy in https://github.com/RedShyGuy/Vapecord-ACNL-Plugin/issues/59#issuecomment-848814738_

Hi, was this fixed? I’m having trouble with this grass editor as well, it works fine, it reloads fine as well, and it saves. Only for the day, as the next day its back to normal with no edits. Cant figure out why and tried saving my backups as well but nothing helps.

**Comment by RedShyGuy** [comment]
You are right, I will check that out, going a day forward does indeed revert the changes

**Comment by RedShyGuy** [comment]
duplicate
